src/lib/datepicker/datepicker.spec.ts

```diff
@@ -1379,13 +1379,6 @@ describe('MatDatepicker', () => {
 
   });
 
-  describe('with missing DateAdapter and MAT_DATE_FORMATS', () => {
```

Comment:
Why was this removed? I would think it would still throw

Reply:
I put it back but referred to the `CdkDatepicker` in the test rather than the `MatDatepicker`.